### ISSUE TITLE
Compound/banked chains

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#88]
In order to get really fine control over what chains can do, we need a 'compound' or 'banked' type of chain to force multiple chains that may have been drawn from different questions to be treated as a single entity. For example:

```python
chains = [
    chain_1,
    chain_2,
    {
        'name': 'chain_3',
        'type': 'banked-chain',
        'text': {'en-GB': 'Top 2 summary q2a-q2c'},
        'views': ['x|mean|x[1,2,3,4,5]:y|||1-5 ^97,98'],
        'items': [
            {'chain': chain_3a, 'text': {'en-GB': 'Top 2 (q5a)'}},
            {'chain': chain_3b, 'text': {'en-GB': 'Top 2 (q5b)'}},
            {'chain': chain_3c, 'text': {'en-GB': 'Top 2 (q5c)'}}
        ]
    },
    chain_3,
    chain_4
]
```

In this case ``chain_3a``, ``chain_3b`` and ``chain_3c`` are the "top 2" boxes from three different links. The expected result when sent to a build is that they all get "banked" together so as to appear as 1 compound table.

**Comment by jamesrkg** [comment]
@AlasdairE this isn't really an improvement to anything except what a build expects a chain to be, so actually this is one for you.

This will open up things like top2 and means summaries, which are already being requested.

The general idea is that the dict overrides what any 1 chain might normally provide (label text, etc.) and each chain in 'items' should be just stacked up one after the other regardless of what link they came from.

What else does this compound chain definition need in order to provide the ExcelPainter with enough information to be workable? Is it possible we can support this soon?

I'm changing the assignee to you because it's essentially a ticket to the ExcelPainter.

**Comment by srinivasgajam** [comment]
Hi @AlasdairE,
Do we have any udpate on this one.

Thanks,

**Comment by jamesrkg** [comment]
Resolved by PR #140
